Procfile

```diff
@@ -1 +1,2 @@
+release: Run migrate -y
```

Comment:
Do we need this explicitly? we've been running auto migrations as far as I aware, so they should just run 🤔 

Me and @adamoxley both made changes that migrated I think